python/lsst/pipe/tasks/postprocess.py

```diff
@@ -281,6 +284,7 @@ class WriteRecalibratedSourceTableConfig(WriteSourceTableConfig,
     )
 
 
+# TODO: deprecate, since reprocessVisitImage does this more thoroughly?
```

Comment:
If you're asking for an opinion, I can see the merit of adding deprecation flags here. If `reprocessVisitImage` can do a better job, I don't see the need to keep this around. With that said, is this beyond the scope of this ticket? 

If deprecations are set up, please also set up the future removal ticket, blocked by the appropriate version.